tests/contrib/pylons/test_pylons.py

```diff
@@ -674,11 +675,36 @@ def test_pylons_body_xml(self):
             assert span
             assert span["mytestingbody_key"] == "mytestingbody_value"
 
-    def test_pylons_body_xml_attack(self):
+    def test_pylons_body_json_unicode_decode_error(self):
+        with self.override_global_config(dict(_appsec_enabled=True)):
+            with override_env(dict(DD_APPSEC_RULES=RULES_GOOD_PATH)):
+                self.tracer._appsec_enabled = True
+                # Hack: need to pass an argument to configure so that the processors are recreated
+                self.tracer.configure(api_version="v0.4")
+                self.app.post(
+                    url_for(controller="root", action="index"),
+                    params=b"\x80",
+                    extra_environ={"CONTENT_TYPE": "application/json"},
+                )
+
+                spans = self.pop_spans()
+                assert spans
+
+                root_span = spans[0]
+                appsec_json = root_span.get_tag("_dd.appsec.json")
+                assert appsec_json is None
+
+                assert (
+                    "UnicodeDecodeError: 'utf8' codec can't decode byte 0x80 in position 0: invalid start byte"
+                    not in self._caplog.text
```

Comment:
```suggestion
                    "UnicodeDecodeError" not in self._caplog.text
```

Should we keep it simple, just in case the error message format were to ever change?


this is a thought, doesn't need to be changed. feel free to resolve without accepting.